modules/contrail-alarm-gen.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, contrailPkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.contrail.alarmGen;
+  confFile = import (./configuration + "/R${contrailPkgs.contrailVersion}/alarm-gen.nix") { inherit pkgs cfg; };
+
+in {
+
+  options = {
+    contrail.alarmGen = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+      };
+      configFile = mkOption {
+        type = types.path;
+        description = "contrail alarm-gen configuration file";
+        default = confFile;
+      };
+      autoStart = mkOption {
+        type = types.bool;
+        default = true;
+      };
+      logLevel = mkOption {
+        type = types.enum [ "SYS_DEBUG" "SYS_INFO" "SYS_WARN" "SYS_ERROR" ];
+        default = "SYS_WARN";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+
+    services.apache-kafka = {
+      enable = true;
+      extraProperties = ''
+        listeners=PLAINTEXT://:9092
+        bootstrap.servers=localhost:9092
```

Comment:
One thing i am wondering about: the kafka server can be configured to any ip address via configuration file but these extraProperties always assume kafka to be running onlocal host?